### PR TITLE
fix images in prod

### PR DESCRIPTION
**Pull request by partha6686**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -155,14 +155,6 @@ export default {
...
   image: {
     dir: 'assets/images',
-    providers: {
-      ipx: {
-        provider: 'ipx',
-        options: {
-          baseURL: 'https://formester-strapi.s3.ap-south-1.amazonaws.com/'
-        }
-      }
-    },
```
Comment (ankitsinghaniyaz) [inline nuxt.config.js:165]: why was this removed?

Comment (partha6686) [reply nuxt.config.js:165]: This was causing the local images to not load.

Comment (partha6686) [reply nuxt.config.js:165]: Just adding the domain works fine. This images are now visible in the preview.

**Review by anirudh-tantry — APPROVED** [review]
